README.md

```diff
@@ -1,4 +1,30 @@
 # ThunderDome
-## Two Goblins Enter One Goblin Leaves ##
+## Two Monsters Enter One Monster Leaves ##
 
 This is a stripped down D&D 5e combat simulator that I am making for practice.
+
+## Build and Run
+
+To build and run this project, first clone it from Github and navigate to the root of the project.  Open a terminal and run the following:
+
+```sh
+# Build
+dotnet build ThunderDome/
+
+# Rin
+dotnet run --project ThunderDome/ThunderDome/ThunderDome.csproj
```

Comment:
Wow this brings back memories.  For some reason Microsoft Projects almost force you to have this goofy `my-project/my-project` structure.  It is possible not to do this but the tools make it look like this is what is required.  Drives me nuts. But this is not that big of deal, just keep your eyes open and see if you can avoid the needless deep folder structure.